tfhe/src/shortint/server_key/tests/noise_distribution/br_dp_ks_ms.rs

```diff
@@ -217,113 +222,21 @@ fn encrypt_br_dp_ks_any_ms_inner_helper(
 
     let before_ms = after_drift.as_ref().unwrap_or(&after_ks);
 
-    match &cks.atomic_pattern {
```

Comment:
@guillermo-oyarzun the thing to simplify decryption stuff you were asking about in your other PR